packages/components/src/PrimaryToolbar/PrimaryToolbar.js

```diff
@@ -160,7 +167,8 @@ PrimaryToolbar.propTypes = {
         actions: Actions.propTypes.actions,
         dropdownProps: Actions.propTypes.dropdownProps,
         onSelect: Actions.propTypes.onSelect
-    })
+    }),
+    expandAll: PropTypes.node
```

Comment:
This could be either `node` or shape`
```JS
expandAll: PropTypes.oneOfType([
  PropTypes.node,
  PropTypes.shape({
    onClick: PropTypes.func,
    isAllExpanded: PropTypes.bool
  })
])
```